pages/resources.js

```diff
@@ -1,15 +1,160 @@
+import Link from "next/link";
+import { useState } from "react";
 import styled from "styled-components";
 
 const Resources = () => {
   return (
     <div>
```

Comment:
You have generated a "Container" styled component in the bottom. Maybe use it here? :)